helpers/metadata_test.go

```diff
@@ -0,0 +1,81 @@
+package helpers
+
+import (
+	tektonv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"testing"
+)
+
+var (
+	testObject      = &tektonv1beta1.PipelineRun{}
+	testAnnotations = map[string]string{
+		"a.test/annotation":              "foo",
+		"another.test/annotation-string": "bar",
+	}
+	testLabels = map[string]string{
+		"a.test/label":              "foo",
+		"another.test/label-string": "bar",
+	}
+)
+
+func init() {
+	testObject.SetAnnotations(testAnnotations)
+	testObject.SetLabels(testLabels)
+}
+
+func TestHasAnnotation(t *testing.T) {
+	checkExistingAnnotation := HasAnnotation(testObject, "a.test/annotation")
```

Comment:
It's ok like this, but just in case you weren't aware about a common patter when testing, you can go through multiple tests with something like:
```
var tests = []struct {
    annotation string
    want          bool
}{
    {"a.test/annotation", true}
    {"foo/bar", false}
}

for _, tt := range tests {
    if tt.want != HasAnnotation(testObject, tt.annotation) {
        t.Errorf("wanted %t for annotation '%s', got %t", tt.want, tt.annotation, !tt.want)
    }
}
```

You can see this pattern for repetitive tests here: https://gobyexample.com/testing

Reply:
Btw, notice that I'm using `Errorf` here to go over every single test case. Fatal would break for the first failure.